setup.py

```diff
@@ -17,7 +17,8 @@
     packages=setuptools.find_packages(),
     install_requires=[
         'omero-py',
-        'numpy'
+        'numpy',
+        'dataclasses'
```

Comment:
This should be 'dataclasses ; python_version < "3.7"', e.g., tell pip to install dataclasses only if python is less than 3.7

Reply:
https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#id5

Reply:
Right. I thought that part was tu plainly enforce python < 3.7 and so match omero's.